### ISSUE TITLE
possible to get option to output [firstname lastname] .bib file instead of only [lastname, firstname]

### ISSUE THREAD

**Issue by emilBeBri** [nschloe/betterbib#181]
Hi, 
thank you for  a great piece of software, has made my life easier.  

I was wondering wether you could implement an option so that an entry edited by betterbib could look like this:
df
```
@article{ocv,
 author = {Ronald L. Breiger and John Doe },
 title = {The Duality of Persons and Groups},
}

```

instead of :

```
@article{chu2016who,
 author = {Chu, Johan S.G. and Davis, Gerald F.},
 title = {Who Killed the Inner Circle? The Decline of the {American} Corporate Interlock Network},
}


```

**Comment by nschloe** [comment]
That's not very "bibtexy". Why would you need that?

**Comment by emilBeBri** [comment]
I don't think you can say that the [latter](url) style is any more bibtexy than the former - it's a matter of convention. overleaf recommends the style that I prefer, for example:

https://www.overleaf.com/blog/532-creating-and-managing-bibliographies-with-bibtex-on-overleaf

I find it alot easier to read at a glanse.

**Comment by nschloe** [comment]
Okay, fair enough. Can you prepare a PR? You're looking at https://github.com/nschloe/betterbib/blob/main/betterbib/tools.py#L182.

**Comment by emilBeBri** [comment]
Sorry, but I'm not proficient in Python (yet - I'm an autodidact R user, Python is on my to-do list for 2021).  If you don't want to spend the time on it, that's quite alright, I'll just change it by hand as I've done so far, better than not having the program is all, that's for sure.

**Comment by nschloe** [comment]
This suggestion hasn't picked up enough interest in a year for me to get active. I'll close it for now.

If anyone else is interested in this, let me know.
